updates/1.124/221006_journalEntry.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<update xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://www.ofw.fi/xml/2011/java-xmldb-updater/UpdaterSchema.xsd">
+
+  <sql>alter table workspaceJournalEntry add column materialFieldReplyIdentifier varchar(255) after created;</sql>
```

Comment:
Could capitalize the W in table name (another case on the next line), some database instances are case sensitive.